src-electron/rest/user-data.js

```diff
@@ -330,17 +330,18 @@ function httpPostCluster(db) {
     let sessionId = request.zapSessionId
 
     try {
-      let packageId = await queryPackage
-        .getSessionPackagesByType(
-          db,
-          sessionId,
-          dbEnum.packageType.zclProperties
-        )
-        .then((pkgs) => pkgs?.shift()?.id) // default to always picking first package
-
-      if (packageId == null) {
-        throw new Error('Unable to find packageId')
-      }
+      let zclPropertiesPkgs = await queryPackage.getSessionPackagesByType(
+        db,
+        sessionId,
+        dbEnum.packageType.zclProperties
+      )
+      let zclXmlStandalonePkgs = await queryPackage.getSessionPackagesByType(
```

Comment:
Instead of multiple queries should we get packages once and then filter based on type? Would get us out of multiple database interactions.